Lib/uuid.py

```diff
@@ -225,6 +237,15 @@ def __init__(self, hex=None, bytes=None, bytes_le=None, fields=None,
         object.__setattr__(self, 'int', int)
         object.__setattr__(self, 'is_safe', is_safe)
 
+    @classmethod
+    def _from_int(cls, int, *, is_safe=SafeUUID.unknown):
+        """Internal use only."""
+        assert int >= 0 and int <= 0xffff_ffff_ffff_ffff_ffff_ffff_ffff_ffff
+        self = cls.__new__(cls)
+        object.__setattr__(self, 'int', int)
+        object.__setattr__(self, 'is_safe', is_safe)
```

Comment:
```suggestion
```
At this moment the argument is unused. Removing it makes the call faster.

Reply:
We still need to create the property, but I can hardcode the value.